build.subr

```diff
@@ -4,23 +4,28 @@
 #
 # Common functions
 #
-# Copyright 2018, 2019 Phoenix Systems
+# Copyright 2018, 2019, 2024 Phoenix Systems
 # Author: Kaja Swat, Aleksander Kaminski, Pawel Pisarczyk
 #
 
+# b_install(src ..., dstdir)
 b_install() {
-	bin="$1"
-	dir="$2"
-
-	if ! [ -f "$bin" ]; then
-		echo -e "\033[0;31m$bin binary does not exist \033[0m"
-		exit 1
-	fi
-
-	echo -e "\033[0;35mInstalling $(basename "$bin") into $dir \033[0m"
-
-	mkdir -p "$PREFIX_FS/root/$dir"
-	install -m 755 "$bin" "$PREFIX_FS/root/$dir"
+	local filelist=("${@:1:$#-1}")
+	local dstdir="${@: -1}"
+
+	local absdir="${PREFIX_FS}/root/$dstdir"
+
+	for file in "${filelist[@]}"; do
+		if ! [ -f "$file" ]; then
+			echo -e "\033[0;31mFile $file does not exist \033[0m"
+			exit 1
+		fi
+
+		if [ "$file" -nt "$absdir/$(basename "$file")" ]; then
+			echo -e "\033[0;35mInstalling $(basename "$file") into $dstdir \033[0m"
+			install -m 755 -C -D -t "$absdir" "$file"
```

Comment:
note: `-D` might not be available on macOS - to be verified

Reply:
Removed all the incremental build stuff for now, so install remains as is